test/test-indexer.spec.js

```diff
@@ -78,10 +80,12 @@ describe('CarIndexer fromIterable()', () => {
     return verifyIndexer(indexer)
   })
 
+  // fails on chrome, not sure why
```

Comment:
even with the changes below? so we need to fix this before we can proceed?

Reply:
Oops, forgot to remove that comment.